src/integrationCapture.ts

```diff
@@ -53,16 +54,19 @@ export const facebookClickIdProcessor: IntegrationCaptureProcessorFunction = (
 // Integration outputs are used to determine how click ids are used within the SDK
 // CUSTOM_FLAGS are sent out when an Event is created via ServerModel.createEventObject
 // PARTNER_IDENTITIES are sent out in a Batch when a group of events are converted to a Batch
+// INTEGRATION_ATTRIBUTES are stored on the event level but are also passed up to the Batch
```

Comment:
can you clarify more what "stored on the event level" means?  It's not stored on events at all, but passed on the batch as you say in the end of the comment